packages/apiserver/src/file/file.controller.ts

```diff
@@ -24,6 +25,15 @@ export class FileController {
     return this.fileService.createFileForTmp(file);
   }
 
+  @Post('/create-from-link')
+  @SkipBlock()
+  @allowUnauthorizedRequest()
+  async createFileFromLink(@Body() body: { file?: string }) {
```

Comment:
What does `fromLink` mean? The argument of the function does no have a `link` but does have a `body: { file }`